src/fetchProjectItems.ts

```diff
@@ -0,0 +1,96 @@
+import { GetProjectsDocument, GetProjectsQuery, GetProjectsQueryVariables } from "./generated/graphql-operations";
+import { ProjectDetails, ProjectItem } from "./types";
+import { print } from "graphql/index";
+import { octokit } from "./octokit";
+
+export async function fetchProjectItems(
+  params: GetProjectsQueryVariables,
+  recursiveCollection?: ProjectDetails
+): Promise<ProjectDetails> {
+  console.log(`Fetching projects`, params)
+  // @ts-ignore
+  const project: GetProjectsQuery = await fetchProject(params).catch(console.error)
+
+  console.log(project)
+
+  if (project.organization?.projectV2?.items?.edges) {
+    const { hasNextPage, endCursor } = project.organization?.projectV2?.items?.pageInfo
+    const items: ProjectItem[] = [];
+
+    project.organization?.projectV2?.items?.edges
+      ?.map(item => item?.node)
+      ?.map(item => {
+        if (item) {
```

Comment:
`item` is a `node` here right? Maybe rename it?

Reply:
[f52d86d](https://github.com/paritytech/roadmap/pull/10/commits/f52d86da25ecb5114aab62186378ad78e85710ee)